### PR TITLE
Skipping PR based on title

### PR DESCRIPTION
**Pull request by KrishMunot**

### CODE REVIEW

**Comment by hunkim** [comment]
Could you add some description about this change? 

I think you need to run tests locally to make sure your change passes all tests.

```
server.js:154
154:   if(skipTitle == "WIP"){
          ^^^^^^^^^ identifier `skipTitle`. Could not resolve name

```

File: server.js [hunk 1]
```diff
@@ -124,6 +124,7 @@ async function work(body) {
     findPotentialReviewers: true,
     actions: ['opened'],
     skipAlreadyAssignedPR: false,
+    skipTitle: "WIP",
```
Comment (vjeux) [inline server.js]: nit: please use single quote:

```
'WIP'
```

File: server.js [hunk 2]
```diff
@@ -150,6 +151,12 @@ async function work(body) {
     return;
   }
 
+  if(skipTitle == "WIP"){
```
Comment (vjeux) [inline server.js]: nit: put a space after `if`, use `===`, single quote `'WIP'` and space before `{`:

```
if (skipTitle === 'WIP') {
```

File: server.js [hunk 3]
```diff
@@ -150,6 +151,12 @@ async function work(body) {
...
+    console.log(
```
Comment (vjeux) [inline server.js]: this fits in one line

File: server.js (hunk 2 above)
Comment (vjeux) [inline server.js]: This logic is not correct. What you want to do is to skip if the title contains WIP. In this case you're going to skip everything if you set the config to WIP

Comment (vjeux) [inline server.js]: skipTitle variable is not defined, did you actually run this code?

**Comment by vjeux** [comment]
Fixed by https://github.com/facebook/mention-bot/pull/130

**Pull request closed without merging**